### PR TITLE
Make standalone tests less verbose

### LINKED ISSUE #11083: Set `enable_progress_bar=False` in standalone tests
## Proposed refactor

The output produced by the standalone tests is unfiltered and produces 1000's of lines due to the progress bar being turned on. This is not needed for 99% - 100% of tests. For regular tests this is not an issue since the tests get batched and the output is not printed in verbose.

### Motivation

Easier to read and scroll the test output logs

### Pitch

Set

`Trainer(enable_progress_bar=False) 
`
for all tests that have

`@RunIf(standalone=True)`

as the marker.
Additionally, could also turn off the model summary.

### Additional context

Happy holidays

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.
``

cc @borda @justusschock @awaelchli @akihironitta

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

The current standalone tests produce many log outputs, around 8700 lines.
This PR sets

```py
enable_progress_bar=False
enable_model_summary=False
```
on all tests that use the Trainer. Reduces logs to < 4000 lines.

Fixes #11083

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @justusschock @awaelchli @rohitgr7 @borda @akihironitta

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]
🚀

**Review by rohitgr7 — APPROVED** [review]

**Comment by awaelchli** [comment]
No, I disagree. This will give people only "ideas" to put other test-specific logic into a TestTrainer.

File: tests/models/test_hooks.py [hunk 1]
```diff
@@ -201,10 +201,11 @@ def train_dataloader(self):
         limit_train_batches=2,
         limit_val_batches=0,
         max_epochs=1,
-        enable_model_summary=False,
         strategy="ddp",
         accelerator="gpu",
         devices=2,
+        enable_progress_bar=False,
+        enable_model_summary=False,
```
[the remark this answers is not in the document]
Comment (justusschock) [reply tests/models/test_hooks.py:208]: I would prefer it to stay as it is. This is more explicit even if it leads to some duplication. And if needed, the tests can be adapted individually without having to either modify the global kwargs or then retype all the kwargs in that specific test

**Review by justusschock — APPROVED** [review]

**Comment by awaelchli** [comment]
gatekeeper merge

**Comment by awaelchli** [comment]
merge

**Review by carmocca — APPROVED** [review]
